client/components/bulk-select/index.jsx

```diff
@@ -54,7 +54,6 @@ export default class extends React.Component {
 				<span className="bulk-select__container">
 					<input type="checkbox" className={ inputClasses } checked={ isChecked } readOnly />
 					<Count count={ this.props.selectedElements } />
-					{ this.getStateIcon() }
```

Comment:
If we are going to remove this we should remove the `getStateIcon` function too.